audit/projects/kubernetes-public/iam.json

```diff
@@ -143,6 +144,12 @@
         "group:k8s-infra-cluster-admins@kubernetes.io"
       ],
       "role": "roles/stackdriver.accounts.viewer"
+    },
+    {
+      "members": [
+        "serviceAccount:k8s-infra-gcp-auditor@kubernetes-public.iam.gserviceaccount.com"
+      ],
+      "role": "roles/viewer"
```

Comment:
Also expected from https://github.com/kubernetes/k8s.io/pull/1974

It's redundant, given a binding to custom org role `audit.viewer` at the org level, but `ensure-main-project.sh` as written needs to give the service account _some_ kind of role at the project level, and it seemed best to avoid tangling with the org for bootstrapping purposes